MakefileHelper

```diff
@@ -39,7 +39,8 @@ pull:
 	rsync -azP ${USERNAME}@${HWSERVER}:${REPORTS_DIRECTORY}/ raw_data/
 
 push:
-	rsync -azP individual_summary_html/* ${USERNAME}@${HWSERVER}:${REPORTS_DIRECTORY}/summary_html/
+	rsync -azP individual_summary_html/*.html ${USERNAME}@${HWSERVER}:${REPORTS_DIRECTORY}/summary_html/
+	rsync -azP individual_summary_html/*.json ${USERNAME}@${HWSERVER}:${REPORTS_DIRECTORY}/seating/final_exam/
```

Comment:
Is this always supposed to be final_exam? Is the user just supposed to know this can be changed?

Reply:
Ideally this would be a command line argument.  For future work....